core/src/main/scala/cromwell/core/actor/BatchingDbWriter.scala

```diff
@@ -69,9 +69,13 @@ trait BatchingDbWriterActor { this: FSM[BatchingDbWriterState, BatchingDbWriterD
   
   def isShuttingDown: Boolean = shuttingDown
   def dbFlushRate: FiniteDuration
+  def dbBatchSize: Int
+  def writeActorName: String
```

Comment:
Cool, I'll try that.  I originally tried `this.getClass.getSimpleName` which did not end well due to trait name mangling, and I wasn't really happy with this solution either but at least it doesn't explode.